stdlib/3/email/message.pyi

```diff
@@ -36,16 +38,16 @@ class Message:
     def get_charset(self) -> _CharsetType: ...
     def __len__(self) -> int: ...
     def __contains__(self, name: str) -> bool: ...
-    def __getitem__(self, name: str) -> Optional[str]: ...
-    def __setitem__(self, name: str, val: str) -> None: ...
+    def __getitem__(self, name: str) -> Optional[_HeaderType]: ...
```

Comment:
Is a `Union` return type a good choice here? See https://github.com/python/mypy/issues/1693 for the issues with returning `Union`.

Reply:
@matthiaskramm what do you think? I'm OK with merging this as is.

Reply:
Alright, let's move forward with this.